### DISCUSSION TITLE
Navigation Parameters when using SelectTabAsync (XF)

### DISCUSSION THREAD

**Discussion by arturgromek** [PrismLibrary/Prism#2444] [Xamarin.Forms]
Hi,

I'm using XF version v4.8 and Prism v7.2, and have a question about using the SelectTabAsync extension method. 

I can pass in the navigation parameters to SelectTabAsync, but how can I access them? 

I am implementing the INavigatedAware and IInitialize interfaces in my base view model. 
The Iniitalize function isn't called with the parameters when the tab is selected, so is there some way to get the values passed when selecting tabs?

I am currently testing on android, and haven't tried iOS, but will need it to work for both.

**Comment by brianlagunas** [comment]
Initialize isn't called when switching tabs. You need to use OnNavigatedTo

**Reply by arturgromek** [reply]
I've tried that as well and OnNavigatedTo isn't fired on my end.

Thanks for the prompt reply :)
